EB docker deploy additions

yinnie opened the pull request:
- Reorganize folder structures so dockerfiles are at root and layers are more flattened
- Add supervisor config, uwsgi config and dockerrunaws.json for multi-docker container elastic beanstalk deploy to work. Based on model from https://github.com/playdots/data-api/pull/4
- Added .envsample and dev.env to put env vars in one place
- Created a RDS instance that uses snapshot of the local data we currently have now in this repo in /mysql_data. (outside of this PR. Reflected in ENV vars in EBS configurations in console)

Deployed to http://td-level-catalog-sandbox.us-east-1.elasticbeanstalk.com/. Needs VPN to acces. Next needs to deploy auth proxy so it is accessible by dots employees.

yinnie commented:
I thought mysql_data is still needed for run dev locally, i.e. using docker-compose?

jonathan-eckel commented:
@yinnie the data is needed to run dev locally, but it shouldn't really be in git.  We can improve PR #1 to initialize a database locally using the raw data in json form.

yinnie commented:
@jonathan-eckel I removed the mysql_data folder (updated .gitignore to ignore it for future) and updated docker-compose to use the locally generated (but gitignored) `/init/mysql_data` folder when running containers locally for development. Updated README too. Your init scripts work well. I just updated AllLevelJsonData using the latest from two dots. We can perhaps make the ipythonnotebook part more executable like you mentioned in the future. So far we have something working!
